src/tn/eduVision/entités/IEntitiy.java

```diff
@@ -4,11 +4,14 @@
  * and open the template in the editor.
  */
 package tn.eduVision.entités;
+
+import java.util.UUID;
+
 /**
  *
  * @author job_j
  */
 public interface IEntitiy {
-    void SetId(int id);
-    int GetId();
```

Comment:
why revertitou ?

Reply:
this is outdated i think . i removed the whole IEntity